crates/pathfinder/src/sequencer.rs

```diff
@@ -3,7 +3,10 @@ pub mod reply;
 pub mod request;
 
 use self::reply::BlockReply;
-use crate::serde::from_relaxed_hex_str;
+use crate::{
+    rpc::types::{BlockHashOrTag, BlockNumberOrTag, Tag},
```

Comment:
Should I move these enums (`BlockHashOrTag, BlockNumberOrTag, Tag`) to some "common_types" module? I don't like the idea of importing an rpc type into the sequencer code.

Reply:
Hmm. My knee-jerk reaction is to say that either

- both need their own implementations (and that they just happen to be the same currently), or
- we stay with importing it (but RPC owns it since that's more "our" code).

I think just keeping it is okay - I also have a bit of an aversion to sharing "same, but not technically the same" types like this. But I think its better than having them float around and belong to no "owner" code.

Reply:
Hmm, let's keep it as is then :thinking: .